napari/__main__.py

```diff
@@ -164,10 +164,14 @@ def parse_sys_argv():
         nargs=0,
         help='show citation information and exit',
     )
+    # Allow multiple --stack options to be provided.
+    # Each stack option will result in its own stack
     parser.add_argument(
         '--stack',
-        action='store_true',
-        help='concatenate multiple input files into a single stack.',
+        action='append',
+        nargs='*',
```

Comment:
```suggestion
        nargs='?',
```
Based on the description of PR there should be one or zero arguments, not any number of arguments.

Reply:
Hello @Czaki, i think there is some confusion about the nargs here. The nargs specifies how many files can be assigned to a Single `--stack` argument:

`--stack img1 img2 img3 ... imgN --stack point1 point2 point3 ... pointN`

I agree that the "clean" way would be to use "+" rather than "*" at the cost of backwards compatibility, since at least one file after `--stack would be required. However `?` would mean zero or one file after stack.

Could you referre to the statement in the PR that lead to the assumption that one or zero arguments should be present? :)

Reply:
I wrong remember argparse. sorry for the wrong comment.

Reply:
No problem :)